Refactor of Tagging Section (#7516)
DataDog/documentation: content/en/logs/log_collection/ruby.md

## Getting further

> ### Connect your service across Logs and APM

Author: ```suggestion
### Connect your service across logs and traces
```

## Getting further

> ### Connect your service across Logs and APM

Reviewer: Let's keep the word `traces`

> If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
> See [APM Ruby logging instructions][7] to learn more about adding trace IDs, span IDs, and even the
> `env`, `service`, and `version` to your logs.

Reviewer: ```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM Ruby instructions][7].
```

> **Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.

Reviewer: ```suggestion
**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.
```
